src/components/ex-download/ex-download.scss

```diff
@@ -0,0 +1,77 @@
+.download {
+  @apply mt-component-spacing-small w-full;
+
+  &__no-title {
+    @apply flex items-center px-4 py-3 font-bold text-white;
+    background-color: rgba(65, 131, 215, 1);
+
+    &:before {
+      @apply mr-2;
+      content: '\26A0';
+    }
+  }
+
+  &_item {
+    @apply flex flex-col w-full px-4 py-2;
```

Comment:
The single download item looks better having .p-4. But then the download item within the list inflate as well. In ideal case a single download has .p-4 and downloads within the download list remain having .px-4.py-3

Reply:
changed the spacing